packages/website/src/App.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react'
+import { StaticQuery, graphql } from 'gatsby'
+import { Flex, Box, Chip, Button, Heading, Text, theme } from '@containous/faency'
+import Icon from 'react-eva-icons'
+import Container from './components/Container'
+import List from './components/List'
+import NavItem from './components/NavItem'
+import Divider from './components/Divider'
+import pkg from '../package.json'
+
+function App({ element, props }) {
+  const [navOpen, setNavOpen] = useState(false)
+
+  const query = graphql`
+    fragment mdxContent on MdxConnection {
+      edges {
+        node {
+          id
+          frontmatter {
+            title
+          }
+          fields {
+            slug
+          }
+        }
+      }
+    }
+    query {
+      pinned: allMdx(
+        sort: { order: ASC, fields: [frontmatter___title] }
+        filter: { frontmatter: { pinned: { ne: null } } }
+      ) {
+        ...mdxContent
+      }
+      components: allMdx(
+        sort: { order: ASC, fields: [frontmatter___component] }
+        filter: { frontmatter: { component: { ne: null } } }
+      ) {
+        ...mdxContent
+      }
+      # recipes: allMdx(
```

Comment:
was this section prefixed by `#` for testing? do you want to keep it or we can remove it?